CONTRIBUTORS.txt

```diff
@@ -78,6 +78,7 @@ Peter Beaman <pbeaman@akiban.com>
 Petter Aas <aaspet@gmail.com> <petter.aas@fronter.com>
 Pieter <pieter@pieter-HP-EliteBook-8570w.(none)>
 PommeVerte <dylan.millikin@gmail.com>
+Rafael Fernandes <luizrafael@gmail.com>
```

Comment:
You should add yourself to `AUTHORS.txt` as well since you retain copyright over your contributions.